mutt_attach.c

```diff
@@ -1149,3 +1148,29 @@ int mutt_print_attachment(FILE *fp, struct Body *a)
     return 0;
   }
 }
+
+/* 2 small functions to handle temporary files */
+
+/* this function just adds a filename to the tail of TempAttachmentsList */
+void mutt_add_temp_attachment(char *filename)
+{
+  struct ListNode *n;
+
+  n = mutt_list_insert_tail(&TempAttachmentsList, NULL);
+  n->data=mutt_str_strdup(filename);
```

Comment:
This function might as well be condensed to just:
```c
mutt_list_insert_tail(&TempAttachmentsList, mutt_str_strdup(filename));
```

It's still clear what's happening.